Automattic/sensei
# Add Lesson Details Step

Fixes #5132

### Changes proposed in this Pull Request

* Rename image used for course details step to a generic name, to be reused by the lesson details step;
* Do some small fixes on the Course Details Step (which I just noticed after the review);
* Work a little bit on the Course Upgrade Step, renaming variables to maintain a pattern and also adding some tests;
* Add the Lesson Details Step;
* And add some tests to the Lesson Details Step;

### Testing instructions

1. Checkout this branch;
2. Run `npm start` to compile the assets (or use the plugin build);
3. Install the plugin in some WordPress installation, access the wp-admin of that WordPress installation;
4. Create a course, and then create a lesson;
5. Access that Lesson;
6. Verify that you can edit the title of the lesson via the wizard that will open;
7. Also verify that the JS tests pass, using `npm run test-js`;
